dev-util/cproto/cproto-4.7e.ebuild

```diff
@@ -1,15 +1,15 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=0
 
-MY_PV="4_7e"
+MY_PV="4.7e"
```

Comment:
this is identical to `${PV}`, or am I missing something?

Reply:
Nice catch, didn't realized this while i was fixing the SRC_URI.. Thanks makes the ebuild even cleaner..